astropy/units/format/console.py

```diff
@@ -30,15 +40,15 @@ class Console(base.Base):
     _space = " "
```

Comment:
It's probably worth annotating these class variables as well.

```suggestion
    _space: ClassVar[str] = " "
```